system_tests/features/steps/get_info_endpoints.py

```diff
@@ -0,0 +1,89 @@
+import requests
+
+from config import Config
+from behave import given, when
+
+
+@given('the ${service} service is running')
+def service_running():
```

Comment:
Suggest changing this to
`def service_running(context):`

Reply:
If the behave framework is designed not to pass context when it's unused it might be best to not define the parameter. It's generally not a good idea to have unused parameters. If we have to have a parameter I think the general pythonic way is to underscore it? `def service_running(_):`

Reply:
Agreed using `_` instead of `context` in this case.